scripts/importLanguages.py

```diff
@@ -1,84 +1,96 @@
+l10n_files = []
 for locale in os.listdir('i18n'):
-    # Skip non folder
+    # Skip non folders
     if not os.path.isdir(os.path.join('i18n', locale)):
         continue
+
     # Skip hidden folders
     if locale.startswith('.'):
         continue
 
-    if PROD and not(locale in PROD_LANGS):
+    # Ignore locale if it's a production build and the locale should not be
+    # included
+    if args.isprod and locale not in prod_locales:
+        continue
+
+    xliff_path = os.path.join('i18n', locale, 'mozillavpn.xliff')
+
+    # If it's the source locale (en), ignore parsing for completeness and
+    # add it to the list.
+    if locale == 'en':
+        print(f'✔\t- en added (reference locale)')
+        l10n_files.append({
+            'ts': os.path.join('translations', f'mozillavpn_en.ts'),
+            'xliff': xliff_path
+        })
         continue
 
-    filePath = os.path.join('i18n', locale, 'mozillavpn.xliff')
-    tree = ET.parse(filePath)
+    tree = ET.parse(xliff_path)
     root = tree.getroot()
 
-    elementCount = 0
+    sources = 0
     translations = 0
 
     for element in root.iter('{urn:oasis:names:tc:xliff:document:1.2}source'):
-        elementCount += 1
+        sources += 1
     for element in root.iter('{urn:oasis:names:tc:xliff:document:1.2}target'):
-        if element.text:
-            translations += 1
+        translations += 1
 
-    complete = translations/(elementCount*1.0)
-    if complete < THRESHOLD:
-        print(f'❌\t- {locale} is translated at {round(complete*100,2)}%, at least {THRESHOLD*100}% is needed')
+    completeness = translations/(sources*1.0)
+    if completeness < l10n_threshold:
+        print(f'❌\t- {locale} is translated at {round(completeness*100, 2)}%, at least {l10n_threshold*100}% is needed')
         continue  # Not enough translations next file please
     baseName = f'mozillavpn_{locale}'
-    print(f'✔\t- {locale} added')
-    FILES.append({
+    print(f'✔\t- {locale} added ({round(completeness*100, 2)}% translated)')
+    l10n_files.append({
         'ts': os.path.join('translations', f'{baseName}.ts'),
-        'xliff': filePath
-    })
-
-if len(FILES) == 0 or len([x for x in FILES if x['ts'] == os.path.join('translations', f'mozillavpn_en.ts')]) == 0:
-    print('No fallback language (en) was imported')
-    os.system(f'lupdate src -no-obsolete -ts translations/mozillavpn_en.ts')
-    FILES.append({
-        'ts': os.path.join('translations', f'mozillavpn_en.ts')
+        'xliff': xliff_path
     })
 
 # Step 2
-# Write PRI file to import the done languages
-with open('translations/translations.pri', 'w') as projectFile:
-    projectFile.write('TRANSLATIONS += \\ \n')
-    for file in FILES:
-        projectFile.write(f"../{file['ts']} \\ \n")
-        os.system(f"touch {file['ts']}")
-    projectFile.write('\n \n##End')
+# Write PRI file to import the locales that are ready
+with open('translations/translations.pri', 'w') as pri_file:
```

Comment:
No functional changes here, it just seems a bit more readable without the newlines.

No clue if the trailing whitespace, or double empty line at the end are needed, just kept the output as it was.